merc_module/mercmodule.py

```diff
@@ -295,17 +372,20 @@ def MakeMoon(whichdir, whichtime):
         # B or D => smaller mass, H or L => larger (j)
         # 1 => smaller axis, 2 => larger axis
         # B or H  => small density, D or L => large density
-        FakeFile = open('FakeMoons.txt', 'r')
+
+        # @TODO: Needs to be made an absolute path?
+        FakeFile = open("FakeMoons.txt", "r")
```

Comment:
We don't actually have this file in the repo and I don't remember what it's for... this whole function is kind of superfluous honestly, as it's not used for the phase of simulations we're trying to run. We can add the absolute path just for uniformity but it shouldn't ultimately matter right now.